types/framework/entities/embeddedEntity.d.ts

```diff
@@ -24,29 +24,29 @@ declare abstract class EmbeddedEntity {
    *
```

Comment:
The `EmbeddedEntity` class is completely addressed in #216, I'd suggest to remove the changes here.